apps/onnx/model.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import model_cpp
+
+
+class Model():
+    def __init__(self):
+        self.pipeline = None
+
+    def BuildFromOnnxModel(self, onnx_model):
+        assert onnx_model
+
+        if type(onnx_model) is str:
+            onnx_model = onnx_model.encode()
+            self.pipeline = model_cpp.ConvertOnnxModel(onnx_model)
+        elif type(onnx_model) is bytes:
+            self.pipeline = model_cpp.ConvertOnnxModel(onnx_model)
+        else:
+            # protobuf don't support swig, so we have to convert them to string.
```

Comment:
doesn't

Reply:
This probably should go away entirely, this is a left over from my initial implementation that was using swig instead of pybind11